modules/37_industry/subsectors/datainput.gms

```diff
@@ -244,7 +244,7 @@ Table pm_calibrate_eff_scale(all_in,all_in,eff_scale_par)   "parameters for scal
 ;
 $offtext
 
-$ifthen.bal_scenario "%cm_import_EU%" == "bal"   !! cm_import_EU
+$ifthen.bal_scenario "%cm_forecast_fix%" == "bal"   !! cm_forecast_fix
```

Comment:
Two points:

1. You may or may not be aware that this switch touches calibration data, so every change of it would require recalibration.  It might therefore be advantageous to include it with `cm_CES_configuration` to make it robust against misconfiguration.
2. Conceptually, this belongs into `mrremind`.  So if this is to stay around for longer, I would recommend to move it there.

Reply:
Pascal works over that and will document switches more clearly. We will generalize some switches in a follow-up process (see [here](https://github.com/remindmodel/development_issues/issues/155)). 

On the ``cm_forecast_fix`` switch. This will be renamed. The purpose of this switch is to fix industrial outputs to a certain trajectory in a policy run. That is, without recalibration. This is because it is not really possible/practical to reach certain outputs in policy scenarios by adjusting the baseline as long as the CES function is quite reactive. It is not the optimal trajectory for REMIND anymore given the baseline, but I think this is not a big issue for projects who don't look at macro effects or mitigation cost between scenarios where this becomes more relevant. Feel free to comment in the above issue on that whether we need it in the future.

Reply:
I agree with Michaja. 
The way it is done right now is not the right way, you either follow one of Michaja suggestions, or use as I mentioned above an endogenous way to assure the alternative target is achieved by "implicit taxes" using for example `cm_implicitQttyTarget `.
I would ask to please create an issue regarding the  `vm_cesIO` fixing and the need to redo this formulation in the future to avoid people using it for other reasons also.

Reply:
Created Issue https://github.com/remindmodel/remind/issues/1033